src/features/events/components/EventChangeCampaignDialog.tsx

```diff
@@ -0,0 +1,206 @@
+        'success',
+        messages.eventChangeCampaign.success({
+          campaignTitle: campaign.title,
+        })
+      );
+
+      router.push(
+        `/organize/${campaign.organization.id}/projects/${campaign.id}/events/${event.id}`
+      );
+
+      handleClose();
+    } catch (error) {
+      setIsLoadingCampaign(0);
+      setError(true);
+    }
+  };
+
+  const handleClose = () => {
+    setIsLoadingCampaign(0);
+    setCampaignFilter('');
+    close();
+  };
+
+  return (
+    <Dialog
+      fullScreen={fullScreen}
+      fullWidth
+      maxWidth={'sm'}
+      onClose={() => {
+        close();
+      }}
+      open={isOpen}
+    >
+      <DialogContent
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          height: '85vh',
+        }}
+      >
+        <Box display="flex" justifyContent="space-between">
+          <DialogTitle sx={{ paddingLeft: 2 }} variant="h5">
+            {messages.eventChangeCampaign.dialogTitle()}
+          </DialogTitle>
+
+          <IconButton onClick={handleClose}>
+            <Close
+              color="secondary"
+              sx={{
+                cursor: 'pointer',
+              }}
+            />
+          </IconButton>
+        </Box>
+
+        <Box display="flex" flexDirection="column" rowGap={1}>
+          <TextField
+            fullWidth
+            id="EventMoveDialog-inputField"
+            InputProps={{
+              startAdornment: <Search color="secondary" />,
+            }}
+            onChange={(ev) => onSearchChange(ev.target.value)}
+            value={campaignFilter}
+            variant="outlined"
+          />
+
+          {error && (
+            <Alert severity="error">
+              <Msg id={messageIds.eventChangeCampaign.error} />
+            </Alert>
+          )}
+
+          <Box
+            sx={{
+              overflowY: 'scroll',
+            }}
+          >
+            <List>
+              {Array.isArray(filteredCampaigns) &&
+                filteredCampaigns.map((campaign) => {
+                  return (
+                    <ListItem
+                      key={`EventMoveDialog-campaignItem-${campaign.id}`}
+                      className={classes.listItem}
+                    >
+                      <Box
+                        alignItems="center"
+                        display="flex"
+                        justifyContent="space-between"
+                        width="100%"
+                      >
+                        <Box alignItems="center" display="flex" marginRight={2}>
+                          <Box marginRight={2}>
+                            <Architecture color="secondary" />
+                          </Box>
+                          {campaign.title}
+                        </Box>
+                        <Box alignItems="center" display="flex">
+                          {!isLoadingCampaign && (
+                            <Button
+                              color="info"
```

Comment:
I know the design has blue buttons, but that's because of a temporary figma problem where the primary color is blue. These buttons should actually use the primary color of our current theme, which is red.

Since `primary` is the default color for buttons, you can just remove this line.

```suggestion
```